.github/workflows/testacc.yml

```diff
@@ -55,7 +55,6 @@ jobs:
           terraform_wrapper: false
       - run: go mod download
       - name: Determine if expensive tests should run
-        if: github.event_name == 'pull_request'
```

Comment:
changing this so we dont run expensive tests on pushes to main if we dont have to